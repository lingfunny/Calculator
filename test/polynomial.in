6
3 2 1 5 8 -3.1 11
3 7 0 -5 8 11 9
+
4 6 -3 -1 1 4.4 2 -1.2 9
4 -6 -3 5.4 2 -1 2 7.8 15
-
7 1 0 1 1 1 2 1 3 1 4 1 5 1 6
2 -1 3 -1 4
+
2 1 1 1 3
2 -1 1 -1 3
+
2 1 1 1 100
2 1 100 1 200
+
3 1 1 1 2 1 3
1 0 0
+