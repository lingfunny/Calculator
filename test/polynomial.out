$-3.1x^{11} + 11x^{9} + 2x + 7$
$-7.8x^{15} - 1.2x^{9} - x + 12x^{-3}$
$x^{6} + x^{5} + x^{2} + x + 1$
$0$
$x^{200} + 2x^{100} + x$
$x^{3} + x^{2} + x$
